src/resources/MachineLearning/ModelInformation/ModelInformation.ts

```diff
@@ -0,0 +1,24 @@
+import API from '../../../APICore';
+import Resource from '../../Resource';
+import {RegistrationModel} from '../MachineLearningInterfaces';
+import {MLModel} from '../Models/ModelsInterfaces';
+import {MLModelInfo} from './ModelInformationInterfaces';
+
+export default class ModelInformation extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/engines`;

```

Comment:
What do you think of this?
```suggestion
    static getBaseUrl = (engineId: string, modelName: string) => `/rest/organizations/${API.orgPlaceholder}/machinelearning/engines/${engineId}/models/${modelName}`;
```
It would simplify the code elsewhere
```
get(engineId: string, modelName: string) {
    return this.api.get<MLModelInfo>(`${ModelInformation.getBaseUrl(engineId, modelName)/details`);
}
```